OpenSim/Simulation/Model/Marker.cpp

```diff
@@ -203,7 +203,7 @@ void Marker::generateDecorations(bool fixed, const ModelDisplayHints& hints, con
     //const Vec3& p_BM = bTrans*get_location();
     appendToThis.push_back(
         SimTK::DecorativeSphere(.005).setBodyId(frame.getMobilizedBodyIndex())
```

Comment:
The tradeoff is either put Sphere(Geometry) as a Component under Marker then it's more flexible, you can change size, or even shape but is verbose and heavyweight that needs connections/attachments to maintain or use generateDecorations route with implicit Geometry which we opted to a while ago to reduce XML impact and redundancy. My preference is to use a single knob (rather than one per decoration type) and allow limited customization.